sca-wrapper-play-30/src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala

```diff
@@ -141,7 +141,8 @@ class WrapperService @Inject() (
       fullWidth = fullWidth,
       hideMenuBar = hideMenuBar,
       disableSessionExpired = disableSessionExpired,
-      optTrustedHelper = optTrustedHelper
+      optTrustedHelper = optTrustedHelper,
+      urBannerUrl = if (urBannerEnabled(bannerConfig)) getUrBannerUrl else None
```

Comment:
Could make this a little more efficient so it doesn't have to iterate thru banner list twice